test/remote/gateways/remote_nmi_test.rb

```diff
@@ -388,6 +388,18 @@ def test_purchase_using_stored_credential_recurring_mit
     assert_success purchase
   end
 
+  def test_purchase_using_ntid_override_mit
+    initial_options = stored_credential_options(:merchant, :recurring, :initial)
```

Comment:
I think to make this test more true to a typical sequence, this initial txn should have `:cardholder` instead of `:merchant`